bbb-common-web/src/main/java/org/bigbluebutton/api/model/request/InsertDocument.java

```diff
@@ -0,0 +1,53 @@
+package org.bigbluebutton.api.model.request;
+
+import org.bigbluebutton.api.model.constraint.*;
+import org.bigbluebutton.api.model.shared.Checksum;
+
+import java.util.Map;
+
+
+public class InsertDocument extends RequestWithChecksum<InsertDocument.Params> {
+
+    public enum Params implements RequestParameters {
+        MEETING_ID("meetingID"),
+        PASSWORD("password");
+
+        private final String value;
+
+        Params(String value) { this.value = value; }
+
+        public String getValue() { return value; }
+    }
+
+    @MeetingIDConstraint
+    private String meetingID;
+
+    @PasswordConstraint
+    private String password;
+
```

Comment:
Actually I think password is not used in this API, in this case it should be removed @GuiLeme

Reply:
Thank you guys, I think that @TiagoJacobs got a point. So I guess I will just extract the password, indeed, since there is no protection from it.

Reply:
After discussing with Tiago it appears that the password in this request is redundant, since a checksum is already being sent, and can be removed. Which leads to questions about the inclusion of password in other requests...